src/components/screens/sprint-view-screen.tsx

```diff
@@ -71,7 +68,8 @@ const SprintViewScreen = () => {
           }); 
           return totalMinutes;
         } catch (error) {
-          console.error(`Error fetching time entries for allocation ${allocation.id}: ${error}`);
+          const message: string = strings.formatString(strings.sprintRequestError.fetchAllocationError, (allocation.id||0).toString(), error as string).toString();
+          setError(message);
           return 0;
```

Comment:
Why are you returning 0 here, if something has caused an error do you have to return anything? If you end up with undefined or something in totalTimeEntries then you could filter out unwanted values before setting it as state.

Reply:
It affects logic of solution. If we are using filtering before, it breaks the indexing. For example, allocation.length will not be equal to timeEntries.length. Here I simplified returns, deleted not necessary ones.